pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -43,35 +49,44 @@ rustPlatform.buildRustPackage rec {
     hicolor-icon-theme
     gsettings-desktop-schemas
     libxkbcommon
-    openssl
-  ];
+  ] ++ (if hasFeature "http" then [ openssl ] else [ ])
```

Comment:
```suggestion
  ] ++ lib.optionals (hasFeature "http") [ openssl ]
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/307726/commits/3567088dfebd8747ee10f028d1fddba07c69bf28, thanks!